mail-client/mutt-wizard/mutt-wizard-20200313.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A system for automatically configuring mutt and isync"
+HOMEPAGE="https://github.com/LukeSmithxyz/mutt-wizard"
+
+COMMIT=5ace69bdf56da3883b2666247aac462b0df5b020
+SRC_URI="https://github.com/LukeSmithxyz/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+notmuch +lynx"
+
+RDEPEND="
+  mail-client/neomutt:=[notmuch=]
+  net-mail/isync
```

Comment:
Is this just a runtime dependency? The README says it's needed for install.

Reply:
I've checked the install script and it is indeed only needed during runtime.

The README needs to be changed.

And indeed, I will add `ssl` to isync.

I really appreciate your help